flow/envs/base_env.py

```diff
@@ -136,10 +128,18 @@ def __init__(self, env_params, sumo_params, scenario):
         # colors used to distinguish between types of vehicles in the network
         self.colors = {}
 
-        # contains the subprocess.Popen instance used to start traci
-        self.sumo_proc = None
+        # create the Flow kernel
+        self.k = Kernel(simulator="traci", sim_params=sumo_params)
+
+        # initialize the simulation using the simulation kernel. This will use
+        # the scenario kernel as an input in order to determine what network
+        # needs to be simulated.
+        self.traci_connection = self.k.simulation.start_simulation(
```

Comment:
er, probably shouldn't be named traci_connection

Reply:
totally agree, but traci_connection is used throughout the codebase, so this is just a temporary fix as we weed them out in later PRs.